modules/page-edit.php

```diff
@@ -162,11 +167,11 @@
 					$page_tags = $_POST['tags'];
 
 				// Insert the "view" part of the page we're editing
-				$content .= parse_page_source($pagetext);
+				$content .=  "<h4>To continue editing or save, scroll down...</h4>" . parse_page_source($pagetext);

```

Comment:
Ah, It's for this message! Thanks for including it. I'd be tempted to use a `<p class="preview-message"><strong>Message here</strong></p>` rather than a `<h4>Message here</h4>`.